src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -13,6 +13,55 @@ public enum JsonCommentHandling : byte
         Disallow = (byte)0,
         Skip = (byte)2,
     }
+    public sealed partial class JsonDocument : System.IDisposable
+    {
+        internal JsonDocument() { }
+        public System.Text.Json.JsonElement RootElement { get { throw null; } }
+        public void Dispose() { }
+        public static System.Text.Json.JsonDocument Parse(System.ReadOnlyMemory<byte> utf8Json, System.Text.Json.JsonReaderOptions readerOptions) { throw null; }
+    }
+    public partial struct JsonElement
+    {
+        private object _dummy;
+        public System.Text.Json.JsonElement this[int index] { get { throw null; } }
+        public System.Text.Json.JsonElement this[System.ReadOnlySpan<byte> utf8PropertyName] { get { throw null; } }
+        public System.Text.Json.JsonElement this[System.ReadOnlySpan<char> propertyName] { get { throw null; } }
+        public System.Text.Json.JsonElement this[string propertyName] { get { throw null; } }
+        public System.Text.Json.JsonTokenType Type { get { throw null; } }
+        public System.Text.Json.JsonElement.ChildEnumerator EnumerateChildren() { throw null; }
+        public int GetArrayLength() { throw null; }
+        public bool GetBoolean() { throw null; }
+        public double GetDouble() { throw null; }
+        public int GetInt32() { throw null; }
+        public long GetInt64() { throw null; }
+        public string GetString() { throw null; }
+        [System.CLSCompliantAttribute(false)]
+        public ulong GetUInt64() { throw null; }
+        public static explicit operator bool (System.Text.Json.JsonElement element) { throw null; }
+        public static explicit operator double (System.Text.Json.JsonElement element) { throw null; }
+        public static explicit operator int (System.Text.Json.JsonElement element) { throw null; }
+        public static explicit operator long (System.Text.Json.JsonElement element) { throw null; }
+        public static explicit operator string (System.Text.Json.JsonElement element) { throw null; }
```

Comment:
This allocates.  We really want an explicit operator for it?  Do we really need any of these casts?

Reply:
The original JsonObject had them, and I left them for... no particular reason.  I'll go ahead and delete them.